src/NULS/Address.cpp

```diff
@@ -0,0 +1,105 @@
+// Copyright © 2017-2019 Trust Wallet.
+//
+// This file is part of Trust. The full Trust copyright notice, including
+// terms governing use, modification, and redistribution, is contained in the
+// file LICENSE at the root of the source code distribution tree.
+#include "Address.h"
+
+#include <TrezorCrypto/ecdsa.h>
+
+#include "../Base58.h"
+#include "../BinaryCoding.h"
+#include "../HexCoding.h"
+
+using namespace TW::NULS;
```

Comment:
```suggestion
using namespace TW;
using namespace TW::NULS;
```
you can remove `TW::` prefix in code